Dockerfile-ubuntu18.04/DockerfilePytorch

```diff
@@ -40,6 +41,11 @@ SHELL ["/bin/bash", "-c"]
 RUN apt-get update -y && apt-get install -y openssh-server
 RUN sed -i 's/#Port 22/Port 12346/g' /etc/ssh/sshd_config
 RUN sed -i 's/#   Port 22/    Port 12346/g' /etc/ssh/ssh_config
-
+# install spark
+RUN wget -qO- https://archive.apache.org/dist/spark/spark-3.2.1/spark-3.2.1-bin-hadoop3.2.tgz | tar xvz -C /home/
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
+ENV SPARK_HOME /home/spark-3.2.1-bin-hadoop3.2
+ENV PYTHONPATH $SPARK_HOME/python/:$PYTHONPATH
+ENV PYTHONPATH $SPARK_HOME/python/lib/py4j-0.10.9.3-src.zip:$PYTHONPATH
```

Comment:
I realized you only installed spark but no hadoop, which leads to we can only use local filesystem instead of HDFS as backend, personally I think it should be OK for now, but we may see this as a potential issue if we need to do distributed data processing.

Reply:
Okay, Thanks.